libs/skills/catalog/frontmcp-development/references/create-plugin.md

```
@@ -287,12 +354,106 @@ class DeleteUserTool extends ToolContext {
+export { MyServiceToken } from './my-plugin.symbols';
+```
+
 ---
 
 ## Official Plugins
 
 For official plugin installation, configuration, and examples, see the **official-plugins** skill. FrontMCP provides 6 official plugins: CodeCall, Remember, Approval, Cache, Feature Flags, and Dashboard. Install individually or via `@frontmcp/plugins` (meta-package).
 
+## Recommended Folder Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)